herbarium-sheet-plant-organ-detection/main.py

```diff
@@ -177,21 +147,23 @@ def run_local(example: str) -> None:
     """
     response = requests.get(example)
     json_value = json.loads(response.content)['data']
-    digital_entity = json_value['attributes']['digitalEntity']
-    additional_info_annotations, width, height = run_object_detection(digital_entity['ac:accessUri'], predictor)
-    annotations = map_to_annotation(digital_entity, additional_info_annotations, width, height)
-    event = {
-        "jobId": str(uuid.uuid4()),
-        'annotations': annotations
-    }
+    digital_media = json_value['attributes']
+    additional_info_annotations, width, height = run_object_detection(
+        digital_media['ac:accessURI'], predictor)
+    annotations = map_result_to_annotation(digital_media, additional_info_annotations,
+                                           width, height)
+    event = map_to_annotation_event(annotations, str(uuid.uuid4()))
     logging.info('Created annotations: ' + json.dumps(event))
 
 
 if __name__ == '__main__':
     cfg = get_cfg()
-    cfg.merge_from_file(model_zoo.get_config_file('PascalVOC-Detection/faster_rcnn_R_50_FPN.yaml'))
-    cfg.merge_from_file('config/custom_model_config.yaml')
+    cfg.merge_from_file(model_zoo.get_config_file(
+        'PascalVOC-Detection/faster_rcnn_R_50_FPN.yaml'))
+    cfg.merge_from_file(
+        '/home/soulaine/IdeaProjects/demo-enrichment-service-image/herbarium-sheet-plant-organ-detection/config/custom_model_config.yaml')
```

Comment:
The config is pointing to local, probably needs to be reverted to `config/custom_model_config.yaml`

Reply:
you mean people won't have a directory called `soulaine` :laughing: 
Thought I caught this before pushing, fixed now